data_registry/admin.py

```diff
@@ -252,6 +252,7 @@ class Media:
                     "active",
                     "status",
                     "context",
+                    "process_notes",
```

Comment:
I'm not sure how this displays, but I think it can be super-long if there are many warnings, etc., so let's put it in its own fieldset, "Logs", after Management.